### ISSUE TITLE
[FEATURE REQUEST] Improve error message in HEMCO when file not found

### ISSUE THREAD

**Issue by msulprizio** [geoschem/geos-chem#98]
Luke Fakes wrote:

>I’ve recently started running GEOS Chem at WACL (York, UK) and have come across an error with a strange error message. It occurs when the BoundaryConditions path is incorrect, the HEMCO.log file provides this information:

```
HEMCO ERROR: Cannot find file for current simulation time: /users/lhf508/scratch/GC/rundirs/geosfp_4x5_tropchem/GEOSChem.BoundaryConditions.17150701_0000z.nc4 - Cannot get field BC_NO. Please check file name and time (incl. time range flag) in the config.
ERROR LOCATION: HCOIO_READ_STD (hcoio_read_std_mod.F90)
ERROR LOCATION: HCOIO_DataRead (hcoio_dataread_mod.F90)
ERROR LOCATION: ReadList_Fill (hco_readlist_mod.F90)
ERROR LOCATION: ReadList_Read (hco_readlist_mod.F90)
ERROR LOCATION: HCO_RUN (hco_driver_mod.F90)
```

>It seems that it looks for the BoundaryConditions for 1715 by default. While the error is easy to fix, it could help to improve the information in the error message.

**Comment by yantosca** [comment]
I've cross-posted this issue on the HEMCO standalone repo as https://github.com/geoschem/HEMCO/issues/2

**Comment by jimmielin** [comment]
Just wanted to add (you may already know this) for info that it is looking for data in `1715` because it cannot find for the current simulation time, and it rewinds a year and again and again until it can find something. This may be particularly problematic as the process is automatic and users to HEMCO may not be aware of this behavior.

It may be useful for HEMCO to enforce strict dates for things like met fields (less so for boundaries, and emissions certainly almost need rewinding years), or at least throw an error message informing the **original** file that fails to be found. This may be present in `HEMCO.log` or not, I am not sure.

**Comment by msulprizio** [comment]
Now fixed in 3c26785fd1225ecf8634acbf225b0e513a0fcde2. This will go into 12.7.0.
